pytest.ini

```diff
@@ -5,3 +5,5 @@ log_cli_level=WARN
 log_format=%(asctime)s %(levelname)s %(message)s
 log_date_format=%Y-%m-%d %H:%M:%S
 junit_family=xunit1
+# ease debugging and fast-fail in CI
```

Comment:
I'm going to be -1 on this, because although it is useful when debugging, one is not always debugging in CI. I've been in situations before when I do need to debug in CI, so I ad-hoc add this to debug, and later I revert the commit.